map:
  cell_size: 0.2 #m
  map_width: 199 #cells
  map_height: 199 #cells
  robot_offset: 0 # offset between robot and the center of the map on the x axis
  rotation: true #rotation of the map 
  lambda_max: 1.0e+5 #max value of lambda
  lambda_unmeasured: 1.0e+5 #default value of lambda in unmeasured cells
sensor:
  radius_filter: 1.2 #m fliter upon the radius of the laser scan  
  angular_filter: 2.094 #rad symetric fliter upon the angular boundaries of the laser scan
  error_region_x: 0.2 #m
  error_region_y: 0.2 #m
  ph: 0.99
  pm: 0.9999
topics:
  laserScan: '/scan' 
  occupancyGrid: '/occupancyGrid'
  massGrid: '/massGrid'
  lambdaGrid: '/lambdaGrid'
  publish_rate: 10 #Hz
tf:
  odom: 'odom'
  lidar: 'laser'
  robot: 'base_link'
normals_estimation:
  neighbor_radius: 20 #neighbors
  max_radius: 2.0 #meters
  min_neighbors: 3 #neighbors

